src/js/Calculator.js

```diff
@@ -0,0 +1,88 @@
+import {Digits} from './components/Digits.js';
+import {Modifier} from './components/Modifier.js';
+import {Operations} from './components/Operations.js';
+import {calculateDisplayText, calculateNumber, isValidSizeOfDigits} from './utils/math.js';
+import {OPERATOR} from './consts/operator.js';
+
+export function Calculator({$el}) {
+
+    const state = {
+        prevNumber: 0,
+        operator: null,
+        nextNumber: null,
+    };
+
+    function onClickDigit({digit}) {
+        renewNumber({digit});
+        refreshTotal();
+    }
+
+    function renewNumber({digit}) {
+        const {prevNumber, operator, nextNumber} = state;
+
+        const newNumber = Number(!!operator
+            ? `${nextNumber || 0}${digit}`
+            : `${prevNumber}${digit}`);
+
+        if (!isValidSizeOfDigits({number: newNumber})) {
+            alert('숫자는 세 자리까지만 입력 가능합니다!');
+            return;
+        }
+
+        if (!!operator) {
+            state.nextNumber = newNumber;
+            return;
+        }
+        state.prevNumber = newNumber;
+    }
+
+    function onClickModifier() {
+        state.prevNumber = 0;
+        state.operator = null;
+        state.nextNumber = null;
+        refreshTotal();
+    }
+
+    function onClickOperation({operator}) {
+        const {prevNumber, operator: prevOperator, nextNumber} = state;
+
+        if (operator === OPERATOR.EQUAL) {
+            state.operator = null;
+            state.nextNumber = null;
+            state.prevNumber = calculateNumber({prevNumber, operator: prevOperator, nextNumber});
+            refreshTotal();
+            return;
+        }
+
+        //이전 연산결과를 prevNumber 저장
+        if (nextNumber) {
+            state.prevNumber = calculateNumber({prevNumber, operator: prevOperator, nextNumber});
+            state.nextNumber = null;
+        }
+        state.operator = operator;
+        refreshTotal();
+    }
+
+    function refreshTotal() {
+        const {prevNumber, operator, nextNumber} = state;
+        const totalDisplayText = calculateDisplayText({prevNumber, operator, nextNumber});
+        $el.querySelector('#total').innerHTML = totalDisplayText;
```

Comment:
문자열만 바뀌는 부분이면 innerText를 쓰시면 더 좋을 것 같아요 
```suggestion
        $el.querySelector('#total').innerText = totalDisplayText;
```

Reply:
https://github.com/next-step/js-calculator/pull/69/commits/47521c790600b2b65c640bc128403b217efa5152 무의식적으로 innerHTML 을 사용했는데 피드백 감사합니다!